handbook/openid-connect/openid-connect.md

```diff
@@ -113,27 +110,31 @@ The `client_id` and `client_secret` can be obtained from the provider. Depending
 
 ### OpenID Configuration and Discovery
 
-The `[provider-dns].provider` file includes the contents of the provider's OpenID Connect configuration. This file informs SystemLink of of the various endpoints exposed by the provider that are used during the login process.
+The `[provider-dns].provider` file includes the contents of the provider's OpenID Connect configuration. This file tells SystemLink which endpoints the provider exposes that are used during login.
 
 !!! note ""
     You may use curl to create this file. Replace `[provider-dns]` with the DNS of your OpenID Connect Provider.
 
-```bash
-curl https://[provider-dns]/.well-known/openid-configuration -o [provider-dns].provider
-```
+    ```bash
+    curl https://[provider-dns]/.well-known/openid-configuration -o [provider-dns].provider
+    ```
 
 ### Setting Login Redirect URI
 
-The client configuration for your provider requires a redirect URL that is used during the login flow. This must by the fully qualified domain name, the protocol (`https://` or `http://`), and the port (if 80 or 443 are not used) of the SystemLink server. If the SystemLink server's DNS changes, this setting must be updated with the provider.
+The client configuration for your provider requires a redirect URL that is used during the login flow. This must by the fully qualified domain name, the protocol (`https://` or `http://`), and the port (if 80 or 443 are not used) of the SystemLink server. If the SystemLink server's DNS changes, update this setting with your provider.
 
-It also must include the port if using a non-default (80/443). We should probably note that it's highly recommended to use the same name for the redirect URI as is set for the preferred host name on the remote tab of the web server config utility.
+!!! note
+    NI recommends the DNS name in the redirect URI match the the preferred hostname set in NI Web Server Configuration on the SystemLink server.
 
 #### The SystemLink login redirect URL
 
-```url
-[protocol]://[systemlink-dns]/login/openidc-redirect
+!!! note ""
+    Use the following URL to configure your provider
```

Comment:
Should this end in a period or colon?
```suggestion
    Use the following URL to configure your provider:
```

Reply:
I added the colon, but I'm curious @jjaco2 's feedback on this.